build.Tests.ps1

```diff
@@ -12,7 +12,7 @@ $expected_base = '../bar/powershellget'
 
 Remove-Item $expected_base -Force -Recurse -ErrorAction Ignore
 
-& $script -PowerShellModuleName "PowerShellGet" -PowerShellModuleVersion "2.1.3"  -OutputDirectory "../bar"
+& $script -ErrorAction Stop -PowerShellModuleName "PowerShellGet" -PowerShellModuleVersion "2.1.3"  -PuppetModuleAuthor 'testuser' -OutputDirectory "../bar"
```

Comment:
We already have `$ErrorActionPreference = "Stop"` in the first line - did that not suffice for here? I'm specifically asking because I'm still fuzzy on wth is going on with error handling in Powershell...

Reply:
hmm my mistake bad rebase ...Let me fix that .Thank you